index.md

```diff
@@ -1,13 +1,116 @@
 # FIAAS
 
+FIAAS is a system that adds PaaS like functionality on top of Kubernetes.
+
+FIAAS provides abstractions for specifying applications and what is required for
+them to run in a given infrastructure.  These abstractions provide the ability
+to separate concerns where application developers can provide a specification
+for their applications and cluster operators can provide specification
+describing the underlying infrastructure hosting the applications. FIAAS enables
+integration of applications with the underlying infrastructure and adapts and
+applies changes to both applications and the infrastructure transparently. This
+allows for seamless reconfiguration and redeployment of all applications if the
+underlying infrastructure is changed and enables applications to be deployed and
+run on infrastructures that are not described or specified in the application.
+This means that an application can be deployed to two very different Kubernetes
+clusters, and still work. In this case, FIAAS acts like an “Interface”, defining
+how the connection should look, and the cluster provides the implementation of
+that “Interface”.
+
+FIAAS has built-in default application configuration options that will apply to
+all applications being deployed. This can be used for driving applications
+towards a common convention and standardization that is sensible. These options
+have built-in defaults but can be adjusted or overwritten as needed on a per
+application basis.
+
+FIAAS is designed with continous delivery in mind and enables connecting
+continuous integration systems with kubernetes clusters for deploying updated
+versions of applications as they become available.
+
+Further information:
+
+* [motivation for creating fiaas](https://github.com/fiaas/fiaas-deploy-daemon/blob/master/docs/fiaas.md).
+* [operator guide](https://github.com/fiaas/fiaas-deploy-daemon/blob/master/docs/operator_guide.md)
+
 ## Getting Started
 
-[Skipper](https://github.com/fiaas/skipper)
+Add the FIAAS helm repository
+
+```
+helm repo add fiaas https://fiaas.github.io/helm
+```
+
+Install Skipper which in turn will bootstrap an instance of Deploy Daemon into
+the default namespace
+
+```
+helm install fiaas/fiaas-skipper --name=fiaas-skipper --set addFiaasDeployDaemonConfigmap="true"
+```
+
+Now there is a Deploy Daemon running that will watch for Application resources.
+
+Apply the application manifest for your application (from your CI system you
+would consume Mast endpoints for generating this).  The application needs to
```

Comment:
Feels like you should reference the overview of components at the bottom here. Mast is being mentioned without any context.

Reply:
👍